tasks/HelpMeCarry.tex

```diff
@@ -0,0 +1,76 @@
+\section{Help Me Carry [Party Host]}
```

Comment:
# Rename!

We have been running *Help Me Carry* in a certain way for years. Changing the entire test content and keeping the name will confuse teams. Either
- Have the robot taking groceries from the car to the kitchen
- XOR Change the name so people won't get confused

Reply:
Carry My Luggage sounds good. How about Roustabout or Drudge?

Reply:
*Carry My Luggage* sounds quite descriptive to me. Let's consider that most participants have a B2-equivalent English proficiency, so uncommon words should be avoided.